docs/pages/router/routing/unmatched.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Unmatched routes
+---
+
+![](/img/unmatched.png)
```

Comment:
Does this file exist?

Reply:
If those images are screenshots or some other visual addition let's use `ImageSpotlight` component instead of displaying raw image:
* https://github.com/expo/expo/blob/main/docs/components/plugins/ImageSpotlight.tsx

Reply:
I couldn't find this file in the https://github.com/expo/router/tree/91ac19c8685c0bdf13496ef7cea47aa1d5bdafe9/docs/docs @marklawlor, is there an image associated with this doc?

Reply:
Nevermind, did found the assets: https://github.com/expo/router/blob/91ac19c8685c0bdf13496ef7cea47aa1d5bdafe9/docs/static/img/unmatched.png